src/main/java/fi/metatavu/dcfb/server/search/searchers/ItemSearcher.java

```diff
@@ -33,19 +34,21 @@
   /**
    * Searches item and returns result as UUIDs
    * 
+   * @param nearLat prefer items near geo point
+   * @param nearLon prefer items near geo point
    * @param categoryIds category ids that must exist on the result. Omitted if null
    * @param locationId location id that must exist on the result. Omitted if null
    * @param search free text search that must match the result. Omitted if null
    * @param firstResult first result. Defaults to 0
    * @param maxResults max results. Defaults to 20
    * @return search result 
    */
-  public SearchResult<UUID> searchItems(List<UUID> categoryIds, List<UUID> locationIds, String search, UUID currentUserId, Long firstResult, Long maxResults, List<ItemListSort> sorts) {
+  public SearchResult<UUID> searchItems(Double nearLat, Double nearLon, List<UUID> categoryIds, List<UUID> locationIds, String search, UUID currentUserId, Long firstResult, Long maxResults, List<ItemListSort> sorts) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Method has 9 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=21&issues=AWVnq7uxXKENQn1Shuht&open=AWVnq7uxXKENQn1Shuht)

[ISSUE_KEY]: AWVnq7uxXKENQn1Shuht